src/Components/RequestGeneratorBuilder.js

```diff
@@ -83,44 +83,40 @@ export default class RequestGeneratorBuilder extends preact.Component {
         // This is a response to a previous request (warning or complaint).
         else if (Privacy.isAllowed(PRIVACY_ACTIONS.SAVE_MY_REQUESTS) && response_to && response_type) {
             new UserRequests().getRequest(response_to).then(request => {
-                fetch(templateURL(this.state.request.language, response_type))
-                    .then(res => res.text())
-                    .then(text => {
-                        this.setState(prev => {
-                            prev.request.custom_data.content = new Template(text, [], {
-                                request_article: REQUEST_ARTICLES[request.type],
-                                request_date: request.date,
-                                request_recipient_address: request.recipient
-                            }).getText();
-
-                            if (response_type === 'admonition') {
-                                // TODO @zner0L: I don't think this is used anywhere.
-                                prev.request.via = request.via;
-                                prev.request.recipient_address = request.recipient;
-                            }
-
-                            prev.request.reference = request.reference;
-                            prev.response_type = response_type;
-                            prev.request.type = 'custom';
-                            prev.response_request = request;
-
-                            return prev;
-                        });
-                        if (response_type === 'admonition' && request.slug) this.setCompanyBySlug(request.slug);
-                        this.renderLetter();
+                fetchTemplate(this.state.request.language, response_type).then(text => {
+                    this.setState(prev => {
+                        prev.request.custom_data.content = new Template(text, [], {
+                            request_article: REQUEST_ARTICLES[request.type],
+                            request_date: request.date,
+                            request_recipient_address: request.recipient
+                        }).getText();
+
+                        if (response_type === 'admonition') {
+                            // TODO @zner0L: I don't think this is used anywhere.
```

Comment:
It is not used *now* but what about the future?

Reply:
Well, I don't even know what this *is*.

But *addressing* != *removing*. If you think this might be useful in the future, leave it and remove the comment instead.